architecture/pagopa.md

```diff
@@ -0,0 +1,203 @@
+# Integration with pagoPA
+
+## Purpose
+
+The purpose of this document is to collect the requirements for integrating
+with the pagoPA payment system. Since the official specs are huge,
+this document is meant to be as a collection of notes about the most important
+elements from the official documents. This document is not meant to be as an
+exaustive specs document but just as a starting point to guide the initial
+phase of the development.
+
+### Non-goals
+
+This document is not meant to define a plan of action.
+
+## Actors
+
+|Actor|Description|
+|-----|-----------|
+|`DC-BE`|Digital Citizenship backend|
+|`DC-FE`|Digital Citizenship frontend (app)|
+|`PPA-BE`|PagoPA backend (_nodo dei pagamenti_)|
+
+## Network requirements
```

Comment:
@gunzip ho parlato con SIA stamattina e suggeriscono che per accedere all'ambiente di test dobbiamo mettere in piedi una VPN site-to-site - magari vediamo se riusciamo a semplificare il setup con una VPN point to site con IP fisso - cmq appena ho i dettagli aggiungo